pkg/hubble/exporter/exporter.go

```diff
@@ -134,15 +131,29 @@ func (e *exporter) Stop() error {
 
 // OnDecodedEvent checks if the event passes the filter.
 // If context was cancelled, it calls Stop() and stops processing events.
-func (e *exporter) OnDecodedEvent(_ context.Context, ev *v1.Event) (bool, error) {
+func (e *exporter) OnDecodedEvent(ctx context.Context, ev *v1.Event) (bool, error) {
 	select {
-	case <-e.ctx.Done():
+	case <-ctx.Done():
 		return false, e.Stop()
```

Comment:
Ha good catch, and I know what happened. The exporter was previously using a stored context received from hubble.launch() instead of the passed in one. Fix incoming, thanks :)

Reply:
Fixed for both exporters in: https://github.com/cilium/cilium/pull/35596/commits/8933ccbb7213db54f3be5ee68868b3c24e1f42ed